flask-server/URLgenerator.py

```diff
@@ -0,0 +1,28 @@
+
+
+import string
+import random
+
+# alphabetLower = list(string.ascii_lowercase)
+# alphabetUpper = list(string.ascii_uppercase)
+# print(alphabetLower)
+# print(alphabetUpper)
+
+alphabet = list(string.ascii_letters)
+
+def get_random_string(length):
```

Comment:
Input should long_url not length, in this case you can put a dummy long url for example "longurl.com" and not use it.

Your output should be the short_url.